priv/repo/migrations/20170201015913_add_group_to_project.exs

```diff
@@ -0,0 +1,11 @@
+defmodule Pairmotron.Repo.Migrations.AddGroupToProject do
+  use Ecto.Migration
+
+  def change do
+
+    alter table(:projects) do
+      add :group_id, references(:groups, on_delete: :nilify_all)
+    end
```

Comment:
As discussed, deleting a group does not delete associated projects (although they will end up in an orphaned state). This is so that users don't lose data on things that they have paired on in the past.